docs/docs/components/count.md

```diff
@@ -21,13 +21,10 @@ The component is **not** displayed while executing the search query or if there
 
   An optional function to override the default rendered component.
 
-- **prefix** `String` _optional_
+- **label** `function` _optional_
 
-  An optional string to be displayed before the number of results.
-
-- **suffix** `String` _optional_
-
-  An optional string to be displayed after the number of results.
+  An optional function to wrap the component with a prefix and suffix string. <br />
+  E.g. `label={(val) => <> Found {val} results</>} />`
```

Comment:
I would probably change the `val` to `cmp` just to indicate that the passed argument is the component not just a value. This would make clear that the return value of the `label` function needs to be a component and cannot be a string :) Please do the corresponding changes to the other places :)